cms/envs/common.py

```diff
@@ -1052,6 +1052,10 @@
 
     # Organizations App (http://github.com/edx/edx-organizations)
     'organizations',
+
+    # Enterprise App (http://github.com/edx/edx-enterprise)
+    'enterprise',
+    'django_object_actions',
```

Comment:
As a matter of principle, do we want `enterprise` to be installed into the CMS? I think not. So maybe we can add a decorator to the affected tests so that they only run in the LMS?